recipe/meta.yaml

```diff
@@ -112,7 +112,7 @@ outputs:
       noarch: python
     requirements:
       run:
-        - {{ pin_subpackage(aiidacore, exact=True) }}
```

Comment:
given the comments, I think removing`exact=True` is not necessary for python extras, since these also have a python version in the build string, so will match

Reply:
No, as you can see here, even those have the build string that does not start with `py` https://anaconda.org/conda-forge/aiida-core.atomic_tools/files

Perhaps because there is no explicit dependency on python

Reply:
i.e from the same build `aiida-core` and `aiida-core.atomic_tools` etc will always have the same build string, but because `aiida-core.services` does not include python it will not

Reply:
I would maybe the put the python dependency in these also, which should then give the same build string

Reply:
Ok, let's try it. I will still remove `exact=True`.

If it turns out that the build strings are identical, we can then reintroduce the pin as before

Reply:
Ok, never say I don't compromise lol